.github/labeler.yml

```diff
@@ -20,11 +22,9 @@ cuGraph:
 CMake:
```

Comment:
No `CMake` label exists in this repo

Reply:
@ajschmidt8 labels are auto-generated if they do not exist; just without color - if the cuGraph team wants this label/path combo they can update the color after this is merged

I see @jolorunyomi removed this, @rlratzel @BradReesWork do you want a `CMake` label?

Reply:
For reference cuML/RMM/cuDF have one for their CMake changes

Reply:
The `cmake` label would be nice since we're making an effort to monitor cmake file changes more.

Reply:
@jolorunyomi can you add this back to match the other repos?

Reply:
the cmake config has been added.

Reply:
Looks good, thanks.